crates/codegen/schema/src/validation/ast/visitors.rs

```diff
@@ -268,9 +267,6 @@ impl Receiver for PrecedenceParserRef {
             for operator in &self.definition.operators {
                 operator.value.receive(visitor, reporter);
             }
```

Comment:
Not sure why `primary_expressions` are excluded from visiting.

Reply:
Because they are references, and if you check the validators you will see they are handled specifically.

Reply:
Not sure what is the purpose of `Visitor::visit_reference` then. I would have expected that validation to go through it.
If I implement `visit_reference()`, I would have expected to get references in primary expressions as well. Am I missing something?

Reply:
No purpose, should have been deleted. A reference is a leaf that should be handled in the node that contains it.

Reply:
done